app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const { getCategories } = require("./Controllers/getCategories.controller");
+const app = express();
+app.use(express.json());
```

Comment:
Does this need to be here yet? If you aren't parsing any of the bodies of your requests then you needn't add it